bot.js

```diff
@@ -1725,14 +1725,32 @@ Actions.findMemberOrUserFromName = async function (name, server) {
 Actions.findMemberOrUserFromID = async function (id, server) {
 	if (!Bot.hasMemberIntents) {
 		PrintError(MsgType.MISSING_MEMBER_INTENT_FIND_USER_ID);
+		return null;
+	}
+	if (!id) {
+		PrintError(MsgType.CANNOT_FIND_USER_BY_ID, id);
+		return null;
+	}
+
+	if (server && id) {
+		let result;
+		try {
+			result = await server.members.fetch(id);
+		} catch (error) {}
+		if (result) {
+			return result;
+		} else {
+			const result = await Bot.bot.users.fetch(id);
+			if (result) {
+				return result;
+			}
+		}
```

Comment:
User fetching is already being done after this piece of code

```suggestion
		const member = await server.members.fetch(id).catch(() => null);
		if (member) {
			return member;
		}
```